### PR TITLE
Change android applicationid 1.0.0

### PR DESCRIPTION
**Pull request by gambit66**
TagCheck

This step allows the updating of an Android applicationId in the apps build.gradle file before the app is built

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [ X] __I will not move an already shared step version's tag to another commit__
- [ X] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [X ] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [ X] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [ X] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

File: steps/change-android-applicationid/1.0.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- build_gradle_path: $BITRISE_SOURCE_DIR/app/build.gradle
+  opts:
+    is_required: true
+    summary: Path to the build.gradle file shows the applicationId setting.
+    title: Path to the build.gradle file
+- new_application_id: com.mycompany.myapp
```
Comment (koral--) [inline steps/change-android-applicationid/1.0.0/step.yml:53]: Default value should be unset.
User should be forced to fill it explicitly, default does not make much sense here imho.

File: steps/change-android-applicationid/assets/icon.svg [hunk 2]
```diff
@@ -0,0 +1,23 @@
+
```
Comment (koral--) [inline steps/change-android-applicationid/assets/icon.svg:1]: nit: extra newline

File: steps/change-android-applicationid/1.0.0/step.yml (hunk 1 above)
Comment (gambit66) [reply steps/change-android-applicationid/1.0.0/step.yml]: Thanks for the review. Default value has been removed.

File: steps/change-android-applicationid/assets/icon.svg (hunk 2 above)
Comment (gambit66) [reply steps/change-android-applicationid/assets/icon.svg:1]: newline removed

File: steps/change-android-applicationid/1.0.0/step.yml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  - [Android Build](https://www.bitrise.io/integrations/steps/android-build)
+  - [Gradle Runner](https://www.bitrise.io/integrations/steps/gradle-runner)
+website: https://github.com/gambit66/bitrise-steps-change-android-applicationId
+source_code_url: https://github.com/gambit66/bitrise-steps-change-android-applicationId
+support_url: https://github.com/gambit66/bitrise-steps-change-android-applicationId
+published_at: 2022-01-21T09:42:55.419371+02:00
+source:
+  git: https://github.com/gambit66/bitrise-steps-change-android-applicationId.git
+  commit: a3d86e36494cc190a3c38d0a9c06966ff58df9f0
+host_os_tags:
```
Comment (mateherber) [inline steps/change-android-applicationid/1.0.0/step.yml:29]: Please remove the `host_os_tags` tag as it is deprecated

File: steps/change-android-applicationid/1.0.0/step.yml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+- react-native
+- cordova
+- ionic
+- flutter
+type_tags:
+- utility
+toolkit:
+  go:
+    package_name: github.com/gambit66/bitrise-steps-change-android-applicationId
+is_requires_admin_user: true
```
Comment (mateherber) [inline steps/change-android-applicationid/1.0.0/step.yml:43]: Please remove `is_requires_admin_user` tag, it is deprecated

**Review by mateherber — CHANGES_REQUESTED** [review]

**Comment by mateherber** [comment]
Thank you for the contribution @gambit66 ! I've added two minor remarks. Please consider applying those changes.

File: steps/change-android-applicationid/1.0.0/step.yml (hunk 3 above)
Comment (gambit66) [reply steps/change-android-applicationid/1.0.0/step.yml]: host_os_tags has been removed

**Comment by gambit66** [comment]
Thanks @mateherber for the review. I have made the requested changes.

**Pull request merged**